internal/pkg/file_watcher/polling_file_watcher.go

```diff
@@ -0,0 +1,101 @@
+package file_watcher
```

Comment:
This is old `file_watcher.go` unmodified (except renaming to PollingFileWatcher)